static/css/main.css

```diff
@@ -29,6 +29,11 @@ code {
   background-color: rgba(0, 0, 0, 0.02);
 }
 
+img {
+  max-width: 100%;
+  height: auto;
```

Comment:
Is it necessary to define `height`?

Reply:
Actually it's not, but these lines are from original hugo theme and I think it's better to not changing them.
but yes it is not necessary.